### PR TITLE
add as_user option, to post the message as the authed user.

### PR DESCRIPTION
**Pull request by lvpython**
post bots message with better bots icon.
from https://api.slack.com/methods/chat.postMessage,
add as_user option.

### CODE REVIEW

File: django_slack/api.py [hunk 1]
```diff
@@ -51,6 +51,11 @@ def slack_message(template, context=None, attachments=None, fail_silently=None,
             'render': ALWAYS,
             'required': NOT_REQUIRED,
         },
+        'as_user': {
+            'default': app_settings.AS_USER,
```
Comment (lamby) [inline django_slack/api.py]: Missing the corresponding entry in ``AppSettings``, no?

(Does tihs code work for you...?)

Comment (lvpython) [inline django_slack/api.py]: ;) You are right, AppSettings updated, it works for now.

**Comment by lvpython** [comment]
Docs done. Thanks, @lamby ;)

**Comment by lamby** [comment]
Many thanks!

**Pull request merged**